website/docs/reference/warehouse-profiles/dremio-profile.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Dremio Profile"
+---
+
+
+:::info Community contributed plugin
+
+This is a Community Contributed plugin for dbt. If you're interested in contributing, check out the source code for the repository [dbt-dremio](https://github.com/fabrice-etanchaud/dbt-dremio)
+
+:::
+
+## Overview of dbt-dremio
+**Status:** Community Contributed
+
+**Author:** Fabrice Etanchaud (Maif-vie)
+
+**Source Code:** https://github.com/fabrice-etanchaud/dbt-dremio
+
+**dbt-dremio** supports dbt 0.18.0 or newer.
+
+When available, the easiest install will be to use pip:
+
+    pip install dbt-dremio
+
+There is no package yet, you can clone the repository, cd in it, and then type 
+
+    pip3 install -e .
```

Comment:
Could you update this to reflect that `dbt-dremio` is now on pypi?

Also, do you plan to match version numbers with dbt minor releases, or keep a different versioning scheme?

Reply:
Yes, I will test the adapter accordingly, the main caveat is that dremio does not have CREATE/DROP SCHEMA commands yet (although there is a PR for that...), I wonder how this will work with the testing framework. I will try and ask for help if I am stuck.

I will follow dbt minor releases.

Thank you @jtcohen6 !

Sorry @jtcohen6 , my english is as good as it should, so could you please explain what you mean by : 

> Only requested change is to acknowledge that you've gotten it onto pypi :)

Reply:
You're all good with the latest commit! I was just repeating myself haha